core/controllers/get_filters.py

```diff
@@ -32,20 +32,18 @@ def get_organisation_names():
 
 def get_funds():
     """
-    Fetches all unique funds sorted alphabetically by fund_type_id.
+    Fetches all funds sorted alphabetically by fund_type_id.
 
-    :return: A tuple - list of unique funds and status code 200. If no funds found, aborts with 404 error.
+    :return: A tuple - list of funds and status code 200. If no funds found, aborts with 404 error.
     """
-    programmes = Programme.query.order_by(Programme.fund_type_id).with_entities(Programme.fund_type_id).distinct().all()
+    funds = Fund.query.order_by(Fund.fund_type_id).with_entities(Fund.fund_type_id).all()
 
-    if not programmes:
+    if not funds:
         return abort(404, "No funds found.")
 
-    funds = [
-        {"name": FUND_ID_TO_NAME[programme.fund_type_id], "id": programme.fund_type_id} for programme in programmes
-    ]
+    fund_list = [{"name": FUND_ID_TO_NAME[row.fund_type_id], "id": row.fund_type_id} for row in funds]
```

Comment:
No specific reason, I've only dealt with the bare bones of fund reference data for this PR because I imagined we'd flesh it out going forward, I can add a ticket to the backlog to look at extra data to add to the column